backend/src/routes/user.ts

```diff
@@ -422,6 +435,37 @@ router.get(
   }
 );
 
+// type UpdateUserType = Infer<typeof UpdateUserCake>
+router.patch(
```

Comment:
Can we move this and the user/me route to a separate file so it doesn't lost with the mentee/mentor routes? Just want to make it more cleaner for future reference.